packages/create-project/commands/init/templates/react/src/App.jsx

```diff
@@ -12,8 +12,7 @@ class App extends Component {
     return (
       <div className="App">
         <h1>
-          Welcome to
-          {name}
+          Welcome to {name}
```

Comment:
Removing the extra line here could make airbnb/eslint complain. Could you instead switch the expression to be:

```jsx
Welcome to
{` ${name}`}
```